packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-expectations.ts

```diff
@@ -215,6 +222,33 @@ function generatedAnswerExpectations(selector: GeneratedAnswerSelector) {
         }
       );
     },
+
+    logGeneratedAnswerFeedbackSubmit(
+      streamId: string,
+      payload: {
+        reason: string;
+        details?: string;
+      }
+    ) {
+      logCustomGeneratedAnswerEvent(
+        InterceptAliases.UA.GeneratedAnswer.GeneratedAnswerFeedbackSubmit,
+        (analyticsBody: {customData: object; eventType: string}) => {
+          const customData = analyticsBody?.customData;
+          expect(analyticsBody).to.have.property(
+            'eventType',
+            'generatedAnswer'
+          );
+          expect(customData).to.have.property(
+            'generativeQuestionAnsweringId',
+            streamId
+          );
+          expect(customData).to.have.property('reason', payload.reason);
+          if (payload.details) {
+            expect(customData).to.have.property('details', payload.details);
+          }
```

Comment:
Should we make our test specific to the current design?

AKA only if `payload.reason === 'other'` can we have `customData.details` ?

Reply:
Yes sure!